### PR TITLE
Adam/geonode integration with PRISM

### PR DESCRIPTION
**Pull request by JorgeMartinezG**
This PR references issues #127 and #128 with the goal of including Geonode/Adam layers within PRISM.

**Changes:**
- Enable the project to work with engines higher than 12.x
- Modify getcapabilities response by removing time values within TIME dimension.
- Include adam layers within layers.json and prism.json files.
- Include serverurl for getcapabilities request.
- Group layers to be queried at the same time. The group is set within the prism.json file, _group_layers_ field.
- Modify reducer with the goal of adding and removing multiple layers using a single toggle button.

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -120,6 +120,6 @@
     "eslint-import-resolver-typescript": "^2.0.0"
   },
   "engines": {
-    "node": "12.x"
+    "node": ">=12.x"
```
Comment (ericboucher) [inline package.json:123]: Is this change necessary or is it to help for local development?
I am generally against having a "broad" version of node in here. If we think we are ready to update to node 14, then we should aim to set the engine to `14.x` instead and fully test end-to-end that this is not breaking anything (build, deploy, ...)

File: src/components/NavBar/MenuItem/index.tsx [hunk 2]
```diff
@@ -12,7 +12,12 @@ import {
 } from '@material-ui/core';
 
 import { MenuItemType, LayerType, TableType } from '../../../config/types';
-import { addLayer, removeLayer } from '../../../context/mapStateSlice';
+import {
+  removeLayers,
```
Comment (ericboucher) [inline src/components/NavBar/MenuItem/index.tsx:16]: Let's try to keep an alphabetical order when possible.

For simplicity, maybe we can just keep the `multiple` version and use `removeLayers([layerId])` if we need to apply it to a single entity?

File: src/utils/server-utils.ts [hunk 3]
```diff
@@ -68,9 +68,10 @@ function formatCapabilitiesInfo(
 
     const availableDates = dates
       .filter(date => !isEmpty(date))
+      .map(date => get(date, '_text', date).split('T')[0])
       .map(date =>
         // adding 12 hours to avoid  errors due to daylight saving
-        moment.utc(get(date, '_text', date)).set({ hour: 12 }).valueOf(),
+        moment.utc(date).set({ hour: 12 }).valueOf(),
```
Comment (ericboucher) [inline src/utils/server-utils.ts:74]: Not a big deal but you could just replace date with `get(date, '_text', date).split('T')[0])` in this line

File: src/components/NavBar/MenuItem/index.tsx [hunk 4]
```diff
@@ -42,6 +47,20 @@ function MenuItem({ classes, title, icon, layersCategories }: MenuItemProps) {
     dispatch(loadTable(table.id));
   };
 
+  const toggleLayers = (layers: LayerType[]) => {
```
Comment (ericboucher) [inline src/components/NavBar/MenuItem/index.tsx:50]: Can you explain a bit more what's happening here? Is this function specific to groupedLayers?

**Review by ericboucher — COMMENTED** [review]
Nice work overall! Maybe make it clear that this PR is specific to Audtralia? (eg. zoom level change + new endpoints)

File: package.json (hunk 1 above)
Comment (JorgeMartinezG) [reply package.json:123]: I had issues setting up locally. No problem removing this constraint

File: src/utils/server-utils.ts (hunk 3 above)
Comment (JorgeMartinezG) [reply src/utils/server-utils.ts]: Done

File: src/config/layers.json [hunk 5]
```diff
@@ -1065,5 +1065,41 @@
         "color": "#0868ac"
       }
     ]
+  },
```
Comment (ericboucher) [inline src/config/layers.json:1068]: If this is specific to Australia, let's make sure we ave a new config for Australia
